assets/js/copybutton.js

```diff
@@ -16,13 +16,15 @@ $(function() {
 
         // For command line code snippets:
         //   1. Strip the prompt from any lines that are prefixed with it
-        //   2. Discard subsequent lines that don't start with the prompt (i.e. output)
+        //   2. Discard subsequent lines that don't start with the prompt (i.e. output),
+        //      making sure to respect line continuation charcters.
```

Comment:
Nit:

```suggestion
        //      making sure to respect line continuation characters.
```

Reply:
Oops, overlooked this, will fix in a separate commit.